### ISSUE TITLE
[2.17] Regression importing from node_modules

### ISSUE THREAD

**Issue by boris-petrov** [ember-cli/ember-cli#7471]
Not sure if this is a regression or I'm doing something wrong, but doing in `ember-cli-build.js`:

```javascript
app.import('node_modules/videojs-contrib-dash/dist/videojs-dash.js');
```

I'm getting with `2.17` the following error:

```
Global error: Uncaught Error: Could not find module `function (f){if(typeof exports==="object"&&typeof module!=="undefined"){module.exports=f()}else if(typeof define==="function"&&define.amd){define([],f)}else{var g;if(typeof window!=="undefined"){g=window}else if(typeof global!=="undefined"){g=global}else if(typeof self!=="undefined"){g=self}else{g=this}g.videojsDash = f()}}` imported from `(require)` at http://localhost:7357/data-server-tests/assets/vendor.js, line 252
```

This happens only with this `videojs-contrib-dash` package. It is the latest version. If I comment out this line, everything works fine. I'm also importing other things from `node_modules` which work fine. This problem didn't happen on `2.16.2`.

Any ideas?

---

Output from `ember version --verbose && npm --version && yarn --version`:
```
ember-cli: 2.17.0                                                                                              
http_parser: 2.7.1                                                                                             
node: 9.2.0                                                                                                    
v8: 6.2.414.44-node.11                                                                                         
uv: 1.17.0                                                                                                     
zlib: 1.2.11                                                                                                   
ares: 1.13.0                                                                                                   
modules: 59                                                                                                    
nghttp2: 1.25.0                                                                                                
openssl: 1.0.2l                                                                                                
icu: 60.1                                                                                                      
unicode: 10.0                                                                                                  
cldr: 32.0                                                                                                     
tz: 2017c                                                                                                      
os: linux x64                                                                                                  
5.3.0                                                                                                          
1.3.2
```

**Comment by rwjblue** [comment]
This might _possibly_ be related to https://github.com/ember-cli/ember-cli/pull/7470, can you check to see?

**Comment by boris-petrov** [comment]
@rwjblue - yes, this fixed the problem. A quick release fix would be nice. :) Thanks for the support!

**Comment by rwjblue** [comment]
Ya, we will get a release out quickly. Just need to get CI fixed and land that other PR...

**Comment by boris-petrov** [comment]
This is now fixed in `2.17.1`. Thank you for the hard work!
